scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -80,6 +80,19 @@ include::modules/ztp-filtering-ai-crs-using-siteconfig.adoc[leveloffset=+2]
 
 include::modules/ztp-configuring-ptp-fast-events.adoc[leveloffset=+2]
 
+include::modules/ztp-add-local-reg-for-sno-duprofile.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-disk-partitioning.adoc[leveloffset=+3]
+
+include::modules/ztp-configuring-pgt-image-registry.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Include ZTP setup process doc here (along with the existing TALM and Image Registry doc)

Reply:
Added "For further information on this, see the topic on ZTP in Additional resources. "

and 

* For more information about Zero Touch Provisioning (ZTP) setup, see href: https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp-deploying-disconnected.html#about-ztp-and-distributed-units-on-openshift-clusters_ztp-deploying-disconnected[About ZTP and distributed units on OpenShift clusters].